services/wallet/activity/service.go

```diff
@@ -101,6 +101,14 @@ func (s *Service) FilterActivityAsync(requestID int32, addresses []common.Addres
 	})
 }
 
+func (s *Service) GetMultiTxDetails(ctx context.Context, id string) (*EntryDetails, error) {
```

Comment:
We should use the original type as much as possible to documenting purpose and type checking. Please change the ID here and in `GetTxDetails`